.github/workflows/autocommit.yml

```diff
@@ -0,0 +1,19 @@
+name: Autocommit pull requests
```

Comment:
What types of PRs is this intended for? Just the project updates from landscape?

Reply:
Yes. Those will receive an `automerge` label when created and so it's clear which ones to automatically merge.